app.js

```diff
@@ -0,0 +1,44 @@
+// DEPENDENCIES
+const express = require("express");
+
+// CONFIGURATION
+const app = express();
+
+// ROUTES
+app.get("/", (request, response) => {
+  console.log("GET request received to route: /");
+  response.send("Welcome 99 Pokemon");
+});
+
+app.get("/jumping/joyous/jellybean", (request, response) => {
```

Comment:
so this isn't dynamic. You want to use "wildcards" to dynamically use paramaters. (i.e: "/:index")